src/CoolPropTools.cpp

```diff
@@ -17,7 +17,7 @@
 #include <stdint.h>
 #include <iostream>
 
-static double ftw_summer; // An evil global variable for the ftw function
+static thread_local unsigned long long ftw_summer; // An evil global variable for the ftw function
```

Comment:
I had to revert this thread_local change since a lot of our compilers (VS2008 especially) don't support C++11 unfortunately.